### DISCUSSION TITLE
Tailpress Themes for Themeforest!

### DISCUSSION THREAD

**Discussion by davidjeba** [tailpress/tailpress#79] [General]
Not even a single Tailpress website is listed in https://themeforest.net/search/tailpress

Can you promote, push the community to list few templates for FREE, Sale etc

Maybe you should run a contest like
"Best Tailpress Template Awards"
and give $100 worth amazon coupon or something for the winning 1-3 prizes

**Comment by jeffreyvr** [comment]
Catchy title. I'm all up for promoting using TailPress, but I don't like the way you bring it forward. Please consider changing the title to something more positive.

**Reply by davidjeba** [reply]
done, hope some positive effect will happen soon :)
